dependencies/vaticle/repositories.bzl

```diff
@@ -20,6 +20,6 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 def vaticle_dependencies():
     git_repository(
         name = "vaticle_dependencies",
-        remote = "https://github.com/vaticle/dependencies",
-        commit = "5be6d949ca1e04e4179ea6acb3432be713b9dfb8", # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_dependencies
+        remote = "https://github.com/alexjpwalker/dependencies",
+        commit = "3244740dd87cb91306fe0dc3418f57ccc640cd0e", # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_dependencies
```

Comment:
Update to `vaticle` once https://github.com/vaticle/dependencies/pull/383 is merged